src/tree.rs

```diff
@@ -355,14 +358,19 @@ impl<'t> MergedNode<'t> {
     }
 
     fn ascii_tree_prefixed(&self, prefix: &str) -> String {
-        match self.merged_children.len() {
-            n if n > 0 => {
-                let children_prefix = format!("{}| ", prefix);
-                let children = self.merged_children
-                                   .iter()
-                                   .map(|n| n.ascii_tree_prefixed(&children_prefix))
-                                   .join("\n");
-                format!("{}📂 {}\n{}", prefix, &self, children)
+        let value_state = self.merge_state.value();
+        let decided_value = value_state.node();
+        match decided_value.kind {
```

Comment:
Let's include the `merged_state` too (value and structure, but I think it implements `Display` already), so we know which side we took when merging.

Reply:
We already do in `impl<'t> fmt::Display for MergedNode<'t>` :)

Reply:
Oh, right, whoops! 😊 Thanks!